src/AppBundle/Entity/BlurayRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+class BlurayRepository extends EntityRepository
+{
+    public function findAll ($page, $maxperpage) {
```

Comment:
findAll is already a repository method (which returns all entities). Use a different method name... Or do not use it (see previous comment)